articles/how-to-optimize-wordpress-website-speed/index.md

```diff
@@ -0,0 +1,214 @@
+The ultimate measure of websites performance is loading speed. This is the amount
```

Comment:
Sorry, I should have been clearer. This occurrence of website should be singular, the second one should be plural.